src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -287,15 +304,21 @@ export default {
             }
 
             this.persist(domainObjectStyles);
+            if (this.stopProvidingTelemetry) {
+                this.stopProvidingTelemetry();
+            }
         },
         updateDomainObjectItemStyles(newItems) {
             //check that all items that have been styles still exist. Otherwise delete those styles
             let domainObjectStyles =  (this.domainObject.configuration && this.domainObject.configuration.objectStyles) || {};
             let itemsToRemove = [];
             let keys = Object.keys(domainObjectStyles);
+            //TODO: Need an easier way to find which properties are itemIds
             keys.forEach((key) => {
                 if ((key !== 'styles') &&
                     (key !== 'staticStyle') &&
+                    (key !== 'defaultConditionId') &&
```

Comment:
we can create a new variable for these conditions?

Reply:
Will do.